src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -251,45 +170,13 @@ private RecordFormat getRecordFormatByJobExecution(JobExecution execution) {
     Future<JobExecution> future = Future.future();
     RestUtil.doRequest(params, RECORD_SERVICE_URL + "/" + record.getString("id"), HttpMethod.PUT, record.encode())
```

Comment:
Maybe we should follow the same approach as with Snapshot entities - define Record type in raml instead of using JsonObject